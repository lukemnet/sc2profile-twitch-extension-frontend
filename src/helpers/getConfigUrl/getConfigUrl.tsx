import getApiUrl from 'src/helpers/getApiUrl/getApiUrl';

const getConfigUrl = (channelId: string) =>
  getApiUrl({ channelId, type: 'config', activity: 'get' });

export default getConfigUrl;
