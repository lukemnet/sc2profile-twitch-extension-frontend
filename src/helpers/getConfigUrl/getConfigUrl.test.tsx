import getConfigUrl from 'src/helpers/getConfigUrl/getConfigUrl';

jest.mock('src/helpers/getApiUrl/getApiUrl', () =>
  jest.fn().mockImplementationOnce(() => ({
    url: 'testConfigUrl',
    method: 'TEST_CONFIG_METHOD',
  })));


it('returns correct config API URL', () => {
  expect(getConfigUrl('123')).toMatchSnapshot();
});
