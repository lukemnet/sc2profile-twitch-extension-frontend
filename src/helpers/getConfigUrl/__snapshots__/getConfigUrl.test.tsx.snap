// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct config API URL 1`] = `
Object {
  "method": "TEST_CONFIG_METHOD",
  "url": "testConfigUrl",
}
`;
